std/traits.d

```diff
@@ -8032,6 +8033,21 @@ private template getSymbolsByUDAImpl(alias symbol, alias attribute, names...)
     static assert(res[0].stringof == "a");
 }
 
+///
+@safe unittest
+{
+    enum Attr;
+    struct A
+    {
+        @Attr void a();
+        @Attr void a(int n);
+              void b();
+        @Attr void c();
+    }
+
+    static assert(getSymbolsByUDA!(A, Attr).length == 3);
```

Comment:
Why don't you check the result directly?
(would be more robust)

A simple way would be `getSymbolsByUDA!(A, Attr).stringof == "tuple(a, a, c)"`